### DISCUSSION TITLE
2023年终总结

### DISCUSSION THREAD

**Discussion by LeetaoGoooo** [LeetaoGoooo/discussion-blog#231] [随笔]
<image src="https://github.com/LeetaoGoooo/discussion-blog/assets/10162120/e22a3c77-fd7b-472e-bf32-7d5c3dbfda57" width="300" />
<image src="https://github.com/LeetaoGoooo/discussion-blog/assets/10162120/55694e8c-2a67-4b70-be80-3ed21367fa02" width="300" />
<image src="https://github.com/LeetaoGoooo/discussion-blog/assets/10162120/c4d57ba4-2e74-4f18-abe4-fbdfcbc6adc3" width="300" />

三张截图图片基本概括了我这一年除工作之外的大部分事情，微信读书截止到 2023.12.23 累计读书 68 本（如果要推荐三本书的话，我应该会推荐：1. 真希望我父母读过这本书；2.失明症漫记；3. 人生有何意义），GitHub 上累计提交 964 次，上架了自己的第一个 App，写了 11 篇博客。到这里总结算是结束了，接下来来谈谈我这一年一些尝试和思考吧。

第一个尝试就是关于自媒体的尝试，今年尝试运营了一个小红书账号，截止到今天粉丝数量为 7.8 k多一点，不过现在基本上放弃了更新，停止运营了，总结下来就是小红书的自媒体环境不太适合程序员，程序员可能更适合 b 站或者油管吧，因为小红书 up 主最终的归宿就是广告或者带货，很难想象程序员可以带货什么或者做什么广告。

第二个尝试就是今年我戒掉了十几年看网络小说的习惯，其实早就注意到网络小说有一定的成瘾性，看过过小说的都知道，一本好的网络小说通常故事情节都很紧凑，引人入胜，代入感极强，时常看的人热血沸腾，在不知不觉间就会侵占很多时间，就像短视频一样。尽管我早就意识到这个问题了，但是去没有行动过，直到今年的某个晚上我躺在沙发上刚看完一本网络小说，有点怅然若失，无所事事，然后随手打开了我的微信读书，突然发现今年已经过去四分之一的时间了，而我并没有读几本书，一瞬间觉得，自己不应该把时间浪费在这些毫无意义的事情上，应该做一些更有意义的事情，然后就把所有的看小说的软件都卸载了，起初我会觉得戒断会是一件很辛苦的事情，没想到进展的异常顺利，从那天晚上之后我就再也没有看过一本电子小说了。

我想说的是，看小说这件事情本身没有对错，有没有意义也是主观的看法，看与不与不看都是自己的选择，**为自己的选择负责就好**。这个事件让我觉得有趣的是，有些我们觉得看起困难的事情，一旦我们付诸行动起来就会发现它并没有我们想象中那么艰难，究其原因大概是**我们渴望成功的同时，更害怕失败，不做就等于不会失败，我们时常会被困在自己给自己设置的围栏之中**。

第三件是关于更科学的精力管理的尝试，很幸运，尝试的结果是好的，这部分大概可以分为四个方面：

1. 	睡眠模式的调整
2. 	锻炼
3. 	饮食
4. 	日常工作精力的管理

为什么会有精力管理的尝试，这是这两年我注意到自己的精力有种大不如前的感觉了，时常会觉得整个人很疲惫，下班之后完全没有再学习和锻炼的精力了，显然可预见如果持续下去这个必然会是一个恶性循环，所以有必要去打破它。

第一个尝试就是对睡眠模式的调整，一方面是作息，一方面是睡眠时间。关于作息，我尝试把 11:30pm - 7:30 am 的睡眠时间往前调整了一个小时，变成了 10:30 pm - 6:30 am，起初会出现睡不着的问题的，但是哪怕睡不着，也坚决不玩手机，坚持了一段时间，慢慢开始适应新的睡眠节奏了，也能感受到早睡早起对自己精力的帮助，感觉白天更有精神了，早起的附带好处就是我在拥有更好的精力的同时，也拥有更多属于可控自己的时间了，晚睡的话，其实时间也有，但是那个时候没有那么多精力让晚上的时间发挥作用，精力下降的同时也会导致自制力的下降。其次，今年也读了几本关于精力管理的书，它们都提到睡眠周期的这个概念，一个睡眠周期为 1.5 个小时，一个人正常 4-5 个睡眠周期就可以了，如果刚好在整数倍的睡眠周期里，就会显得很有精神，所以我的睡眠时间从之前的每天8个小时，变成了 6 到 7.5 个小时，基本上如果每天早上醒来到时间在 6 点左右，我就会选择起床。

然后关于锻炼，再次从早上锻炼切换到晚上锻炼，就我个人而言，我发现早上锻炼在一定程度上消耗了休息一晚上回复的精力，会让我觉得一天下来更加疲惫，所以我选择了晚上锻炼，锻炼完不久就可以洗漱睡觉了，锻炼也会一定程度上释放掉白天一天的工作压力，为了不让锻炼影响我的睡眠，我通常也会在睡觉前的 2-3 个小时前去锻炼，也就是 7:30pm-8:30pm 这个时间，这样子不至于让自己过于兴奋从而导致失眠；运动方式从单一的游泳或者跑步，开始多样化了一些，加入力量训练，骑行，乒乓球，以及羽毛球等，让运动变得更有趣也更容易坚持了一些。

接着是关于饮食，身边的朋友都知道我的做饭风格一直是少油少盐，早餐基本上都是自己在家做，一般都是包子（后半年变为粗粮），鸡蛋，牛奶，水果，后半年又加入了坚果，补充一些优质脂肪；一天三顿份量的话，都保持在 7 份饱左右，不会让自己吃撑，下半年得益于新公司的办公的灵活度，点外卖点次数大幅度下降，在家做饭吃饭的频率增加，除此之外，8:30 pm 之后基本不再会吃额外的东西，含糖饮料和零食自然也是少之又少。

最后是日常工作的精力管理，首先明确了一点就是**人的精力是有限的，专注力是需要需要消耗精力的，错误的放松方式同样会消耗掉精力**。从我自己的实践可以感知到一个现象就是一直从事某个事件，随着时间的拉长，单位时间消耗的精力也会增加，需要摆脱这个问题，就需要短暂的休息，也就是说在工作一段时间后，你的精神会进入一定的疲惫期，你需要找个恰当的方式去让它休息一下，就像橡皮筋一样，如果一直让它紧绷着，哪怕后面松手它也很难恢复到之前最初的状态。需要注意的是，刷手机并不会帮助你恢复精力，它只会让你觉得更累，有点饮鸩止渴的感觉，正确的方式应该是从座位上站起来，起身接杯水，溜达一下，然后呼吸一下新鲜的空气，和同事聊会天，眺望远处，看会书等等。

第四件就是关于读书这件事情的思考。**当某种行为成为了下意识的习惯，坚持就不再是难事。大脑总是倾向于去做一些不太费力的事情，不怎么需要决策的事情，而恰巧的是，习惯就是这样子的。但是如果一直被习惯推动着很难说是一件特别正确的事情，毫不费力的事情总是鲜有思考的**。我今年感觉自己有点迷失了，迷失读书的习惯之中，一本接着一本，被所谓的数字游戏玩弄，追求时间，追求数量。书是越读越多，读书的时间也越来阅读，但是我的思考却在变的越来越少了。因为思考总是要耗费功夫的，这也是我在偷懒的一个表现—逃避思考。在新的一年，我准备在每本书读完之后，留个自己一个思考的时间，花点时间认认真真地写点东西，我的思考时间不应该比读一本书的时间相差太多，甚至我觉得应该要超过读书所花的时间。

第五件也是最后一件，**做自己喜欢的事情，做自己认为是对的事情，剩下交给时间**。如果你自己做的事情同事具备喜欢和对的事情，不需要在意别人的看法，喜欢和对本来就是一个主观性极强的事情，别人会有不同的看法也是在所难免的，做自己喜欢并且认为对的事情最大好处就是，你会充满热情，有着克服困难的勇气。

# 精力管理参考书籍

- 效率脑科学：卓有成效地完成每一项工作
- 纳瓦尔宝典
- 加油吧，大脑！

**Comment by Jsaeron** [comment]
看过留名~加油。

**Reply by LeetaoGoooo** [reply]
哈哈 加油～
